webonary-cloud-api/lambda/__tests__/databaseSetup.ts

```diff
@@ -15,7 +15,7 @@ export function setupMongo(): void {
 
     const dbClient = await connectToDB();
     const db = dbClient.db(MONGO_DB_NAME);
-    await createIndexes(db);
+    await createReversalsIndexes(db);
```

Comment:
should this also setup entry indices?

Reply:
Ah, good point! We do not have special search logic tests that would need these indexes. Will need to do so in the future. Thanks for the reminder.